projects/dotcms-field-elements/src/utils/utils.tsx

```diff
@@ -31,6 +31,8 @@ export function getClassNames(
  * @returns {DotOption[]}
  */
 export function getDotOptionsFromFieldValue(rawString: string): DotOption[] {
+    rawString = rawString.replace(/(?:\\[rn]|[\r\n]+)+/g, ',');
```

Comment:
From the backend the value for the options is coming like: `'Canada|CA\r\nMexico|MX\r\nUSA|US'` so this is for supporting that, I update unit testing.